pymc3/stats.py

```diff
@@ -151,16 +150,18 @@ def waic(trace, model=None, n_eff=False, pointwise=False):
 
     p_waic = np.sum(vars_lpd)
 
-
-    if n_eff:
-        return waic, waic_se, p_waic
-    elif pointwise:
-        return waic, waic_se, waic_i, p_waic
+    if pointwise:
+        return pd.DataFrame([[waic, waic_se, p_waic, waic_i]],
```

Comment:
@twiecki still thinking namedtuple is the way to go? If so let me now and I will change it.